pages/_app.js

```diff
@@ -28,15 +28,17 @@ function MyApp({ Component, pageProps, err  }) {
   })
 
   return <>
-    <Script id="google-tag-manager" strategy="afterInteractive">
-      {`
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','GTM-WGX2SMF2');
-      `}
-    </Script>
+    { process.env.ANALYTICS_ENABLED === 'true' && (
```

Comment:
Oh, I guess it should be a string.

I'd instead make `ANALYTICS_ENABLED` a boolean, since you're parsing it in the next config anyway. And then simply do:
```ts
{ process.env.ANALYTICS_ENABLED && () }
```

Reply:
Agreed https://github.com/LBHackney-IT/housing-repairs-online-frontend/pull/82/commits/2051e2ff9e2b2675188e8e2a443752b69446f09c